src/pages/auth/reset-password/[token].page.tsx

```diff
@@ -44,10 +45,12 @@ const ResetPasswordPage = () => {
       if (!router.query.token) {
         router.push("/");
       } else {
-        await requestResetPassword({
+        resetPassword({
```

Comment:
Please don't call v3 API methods directly from components. The ResetPasswordConnection should accept `token` as a prop, and expose a `resetMethod` function that just takes the new password and knows how to apply the password to the body and the token to the URL. Component code shouldn't have to know anything about how things are being updated, but should simply have access to methods for modifying data while the Connection code handles the specifics.

As mentioned above, the connection needs the `token` as a prop anyway for `isLoading` and `requestFailed` to work correctly.